python/sglang/srt/models/deepseek_v2.py

```diff
@@ -283,6 +287,22 @@ def __init__(
                     else {}
                 ),
             )
+            if self.shared_experts.gate_up_proj.weight.dtype == torch.int8:
+                assert self.shared_experts.down_proj.weight.dtype == torch.int8
```

Comment:
can we skip the checks here? gate_up_proj.weight.dtype should be the same as gate_down_proj.weight.dtype, do we have exceptions?

if we can skip the checks, the code can be further simplified.

Reply:
Yes they should be the same. Let me remove this check.

Reply:
Updated the code in https://github.com/sgl-project/sglang/pull/6769/commits/ab237a979cd2b9693dac79ced712a7ecf6f8217b.
Could you check if the updated code is good now?